Add the SC update for March 2022 (#120)
python/steering-council: updates/2022-03-steering-council-update.md

* The SC accepted PEP 675 (Arbitrary Literal String Type).
* The SC discussed PEP 678 (Enriching Exception with Notes), and decided to wait for another round of discussions.
* The SC discussed giving a keynote at PyCon US.
* The SC agreed that PSF should fund a second Developer in Residence.
* The SC briefly discussed the "Contributing Cinder" document from Meta, and agreed the first steps should be handled in individual issues and pull requests.
* The SC discussed PEP 670 (Convert macros to functions in the Python C API).
* The SC discussed PEP 655 (Marking individual TypedDict items as required or potentially-missing) and typing PEPs in general.

## 2022-03-14

* The SC met with the Developer in Residence to discuss GitHub issues migration, Language summit invitations, PyCon, release management, and extending the Developer in Residence contract.
* The SC discussed PEP 678 (Enriching Exception with Notes), and found it still too complex. Questions and suggestions to go on the Discourse thread.
* The SC accepted PEP 655 (Marking individual TypedDict items as required or potentially-missing).
* The SC decided to ask typing-sig for delegates, and to convert PEPs to specifications.

> * The SC discussed finding a release manager for 3.11-3.12.

Reviewer: [Pablo is doing 3.10-3.11](https://devguide.python.org/#status-of-python-branches), should this be for the next pair?

```suggestion
* The SC discussed finding a release manager for 3.12-3.13.
```

* The SC discussed gathering information on typing annotation PEPs (564, 649).

## 2022-03-21

* The SC discussed the recent 3.9/3.10 security releases.
* The SC discussed finding a release manager for 3.11-3.12.

> * The SC agreed on the acceptance notices for PEPs 665 and 675.

Reviewer 2: Are these numbers right? The notes claim 675 was accepted on 3/7, and 665 was rejected quite a while ago.

Author: It's 655, thanks for the catch!
It took a while to get the message sent after acceptance.

```suggestion
* The SC agreed on the acceptance notices for PEPs 655 and 675.
```

* The SC discussed PEP 678 (Enriching Exception with Notes), and decided to wait for more discussions. 
* The SC discussed 2-factor authentication for the CPython repo.
* The SC discussed the future of the Developer in Residence role.
* The SC briefly discussed the PyCon keynote.

## 2022-03-28

> * The SC met with the Developer in Residence to discuss the delayed GitHub issues migrationCLA bot, PyCon keynotes, and 3.9/3.10 security releases.

Reviewer 3: Is there a missing comma and space here, between "GitHub issues migration" and "CLA bot"?

Reviewer 4: ```suggestion
* The SC met with the Developer in Residence to discuss the delayed GitHub issues migration, CLA bot, PyCon keynotes, and 3.9/3.10 security releases.
```
